content/changelog-stable-old/index.html.haml

```diff
@@ -0,0 +1,28 @@
+---
+layout: changelog
+title: Archived LTS Changelogs
+show_ratings: false
+has_rss: false
+actions:
+  <a class="app-button app-button--primary app-mobile-hide" href="/changelog-stable">
+    <ion-icon name="arrow-back-circle-outline"></ion-icon>
+    Back to Recent LTS Changelogs
+  </a>
+---
+
+.ratings
+  - # Load older changelogs (all except the most recent 25, ordered correctly)
+  - archived_changelogs = site.changelogs[:lts].reverse.drop(27)
```

Comment:
Is there a reason why the `27` is hard-coded?

Reply:
And, why 27?

Reply:
Hi Sir @krisstern  ,I observed that using 25 excluded the January 2023 releases from the correct page when I saw in local build, so I adjusted it to 27 to include them. I hard-coded 27 as a  fix to show latest 2 years list, @MarkEWaite Sir told me about this that instead of date computations ,release of counts are used. Let me know your thoughts if it is fine or can be improved !

Reply:
I don't think that was exactly what Mark said. Direct quote:
>  The weekly changelog archive (changelog/ and changelog-old/) avoid date computations by using a count of releases instead of a date. If you use a count of 24 or 25 releases, that is approximately two years of LTS releases (a release every 4 weeks with a two week break at the end of each calendar year).

The issue is I don't know why we have the number 25 before, but then you seem to need 27...

Reply:
I tend to think we should keep 25... unless the other maintainers have a different view

Reply:
Hi @krisstern ,

Thank you for pointing this out. I understand now that the original intent was to approximate two years of LTS releases, which aligns with using 24 or 25 releases. My change to 27 was primarily to ensure that the January 2023 releases were included in the list during my local testing since then we could take exact two years starting from 2023.

Based on your feedback, I’ll revert this back to 25 for consistency Or I should wait for other maintainer view too Sir?

Let me know if you have any additional thoughts or suggestions!